Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1427,11 +1427,10 @@ class BrowserViewController: UIViewController {
             ]
             
             SecTrustSetPolicies(serverTrust, policies as CFTypeRef)
-            SecTrustEvaluateAsync(serverTrust, DispatchQueue.global()) { _, secTrustResult in
-                switch secTrustResult {
-                case .proceed, .unspecified:
+            SecTrustEvaluateAsyncWithError(serverTrust, DispatchQueue.global()) { _, secTrustResult, _ in
```

Comment:
This line causes a bug (undefined behaviour) to happen on a very rare occasion which causes the result to fail evaluation or take a long time for evaluation to complete (only with the new iOS 13+ API). Why? Apple recently changed the documentation instead of fixing the bug to say: 

> You must call this method from the same dispatch queue that you specify as the queue parameter.

So to fix it, you need to do:
```swift
let queue = DispatchQueue.global()
SecTrustEvaluateAsyncWithError(serverTrust, queue) { _, secTrustResult, _ in
    ....
}
```

Reply:
Lol why would Apple ask for a queue to execute on for an _async_ method then require you to also call it from that queue 😂🤦‍♂️ Will fix that.

Do we know what websites that would trigger on? We could test and see what error gets thrown and perhaps act accordingly?

Reply:
Finally fixed this!

Reply:
I found the Apple change for certs: https://support.apple.com/en-us/HT210176 and https://support.apple.com/en-us/HT211025

IIRC, when I used the older API, it allowed sites with certs that didn't match the new requirements to work at the time but I just checked and it seems it is now an OS level thing and not just an API thing so we're good to go. IE: Using the old API vs. new one no longer has any difference on certificate validation. Not sure when that changed but it must've been a bug and got patched.

Reply:
Awesome, thanks for looking into this 🙂